### PR TITLE
Bundling typescript definition file

### PR DESCRIPTION
**Pull request by varshard**
Bundling a typescript definition file for omise-node.

Related issue:
[issue 58](https://github.com/omise/omise-node/issues/58)

### CODE REVIEW

**Comment by iporsut** [comment]
@wingyplus Please review 🙇

File: types/examples/create_customer.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  publicKey: process.env.OMISE_PUBLIC_KEY,
+  secretKey: process.env.OMISE_SECRET_KEY
+});
+
+const customer = {
+  email: 'john.doe@example.com',
+  description: 'John Doe (id: 30)'
+};
+
+omise.customers.create(customer, function(err, resp) {
```
Comment (turboza) [inline types/examples/create_customer.ts]: arrow

File: types/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,431 @@
...
+    }
+
+    interface Request {
+      description?: string;
+      amount: number;
+      currency: string;
+      capture?: boolean;
+      card?: string;
+      customer?: string;
+      return_uri?: string;
```
Comment (turboza) [inline types/index.d.ts]: `metadata` is missing na
https://www.omise.co/store-your-data-on-omise-with-the-metadata-api

Comment (varshard) [reply types/index.d.ts]: Fixed.

**Comment by wingyplus** [comment]
@varshard @iporsut  Should we rebase master on this branch to make CI pass?

**Comment by wingyplus** [comment]
cc @turboza

**Comment by varshard** [comment]
@wingyplus My mistake. I renamed every occurrence of some TS's interfaces, but look like some slipped into js code.

**Comment by wingyplus** [comment]
@varshard Thanks. Can you review #71 and #72? It's PR compare to `typescript` branch to make typescript definition work better. :)

**Pull request merged**